platform/api/openstack/api.go

```diff
@@ -628,5 +646,21 @@ func (a *API) GC(gracePeriod time.Duration) error {
 			return fmt.Errorf("couldn't delete server %s: %v", server.ID, err)
 		}
 	}
+
+	images, err := a.listImagesWithTags([]string{"mantle"})
+	if err != nil {
+		return fmt.Errorf("listing Mantle images: %w", err)
+	}
+
+	for _, image := range images {
+		if image.CreatedAt.After(threshold) {
+			continue
+		}
```

Comment:
I just noticed that the similar thing could also be achieved with `ListOpts.CreatedAtQuery`.

Reply:
Cool, this can be include in a follow-up PR, for example this whole section could be simplified: https://github.com/flatcar-linux/mantle/blob/9fc1820fc76db08a7670b7b59c75c0c4b87e5561/platform/api/openstack/api.go#L585-L611